### PR TITLE
feat(asyncwrapper): more types to string

### PR DESCRIPTION
**Pull request by hugosenari**
To be honest, I could only test CASS_VALUE_TYPE_UUID

### CODE REVIEW

File: cassandra/asyncwrapper.nim [hunk 1]
```diff
@@ -447,6 +452,28 @@ converter toBool*(v: Value): bool =
...
+    result = Duration(months: months, days: days, nanos: nanos)
+
+converter toCassUuid*(v: Value): CassUuid =
+    var u: CassUuid
+    chck cass_value_get_uuid(v.o, addr u)
+    result = u
+
+
+proc `$`*(d: Duration): string =
+    "P" & $d.months & "M" & $d.days & "DT" & $d.nanos & "N"
```
Comment (hugosenari) [inline cassandra/asyncwrapper.nim]: Long version should be something like
```nim
  var 
    month: int32
    days:   int32
    nanos:  int64
  discard v.o.cass_value_get_duration(addr month, addr days, addr nanos)
  let year  = month div                12
  month     = month mod                12
  let hours = nanos div 3_600_000_000_000
  nanos     = nanos mod 3_600_000_000_000
  let mins  = nanos div    60_000_000_000
  nanos     = nanos mod    60_000_000_000
  let secs  = nanos div     1_000_000_000
  nanos     = nanos mod     1_000_000_000
  let mili  = nanos div         1_000_000
  nanos     = nanos mod         1_000_000
  let micr  = nanos div             1_000
  nanos     = nanos mod             1_000
  # ISO8601 Durations https://en.wikipedia.org/wiki/ISO_8601#Durations
  "P" & $year & "Y"  & $month & "M"  & $days  & "DT" &
       $hours & "H"  & $mins  & "M"  & $secs  & "S" &
       $mili  & "ms" & $micr  & "us" & $nanos & "ns"  #  there is no ms/us/ns in ISO :(
```

File: cassandra/asyncwrapper.nim [hunk 2]
```diff
@@ -30,6 +30,11 @@ type
         o*: ptr CassValue
         gcHold: GcHoldRef
 
+    Duration* = object
+        months*: int32
+        days*: int32
+        nanos*: int64
+
```
Comment (yglukhov) [inline cassandra/asyncwrapper.nim]: Can we reuse existing `times.Duration` here?

Comment (hugosenari) [reply cassandra/asyncwrapper.nim]: Options:
1. Use times.Duration and fixed month in 30
  - :+1: Simple
  - :-1: Imprecise (there is no month in times.Duration)
2. Use TimeInterval.
  - :+1:  Precise
  - :-1: Conversion is ugly
3. Keep this obj
  - :+1: Simple
  - :-1: :-1:  Too much glue code for lib users to integrate with std

File: cassandra/asyncwrapper.nim [hunk 3]
```diff
@@ -447,6 +447,40 @@ converter toBool*(v: Value): bool =
...
+    nanos     = (nanos  mod     1_000_000_000)
+    let mili  = (nanos  div         1_000_000).int
+    nanos     = (nanos  mod         1_000_000)
+    let micr  = (nanos  div             1_000).int
+    nanos     = (nanos  mod             1_000)
+    result = initTimeInterval(
+      years = years, months = months,
+      hours = hours, minutes = mins, seconds = secs,
+      milliseconds = mili, microseconds = micr, nanoseconds = nanos.int
+    )
```
Comment (hugosenari) [inline cassandra/asyncwrapper.nim:471]: I told you that this is ugly
After so many `mod`, I would not expect it to overflow :crossed_fingers:

**Comment by hugosenari** [comment]
SELECT * FROM system.local


```csv
key    ,bootstrapped,broadcast_address,cluster_name ,cql_version,
"local","COMPLETED" ,11.111.11.11     ,"cluster_stg","3.4.4"
```
```csv
data_center,gossip_generation,host_id
"dc1"      ,1692745003       ,5d550132-68d5-4e09-ad6c-9aa7b31ea445
```
```csv
listen_address,native_protocol_version,partitioner
11.111.11.11  ,"4"                    ,"org.apache.cassandra.dht.Murmur3Partitioner"
```
```csv
rack   ,release_version,rpc_address ,schema_version
"rack1","3.11.2"       ,11.111.11.11,153fa4d5-fd0f-3828-aaa6-8fdbd012bf3b
```
```csv
thrift_version,tokens,truncated_at
"20.1.0"      ,?SET? ,?MAP?
```

"SELECT now(), toDate(now()) FROM system.local"
```csv
system.now()                        ,system.todate(system.now())
f1e4d9a0-5965-11ee-98df-ed4e8e794594,2147503270
```

**Comment by yglukhov** [comment]
Thank you

**Pull request merged**